### PR TITLE
[HttpKernel] Explain how to define default value in MapQueryString

### PR DESCRIPTION
**Pull request by NeilPeyssard**
Hello,

The current example of how using a `MapQueryString` attribute is not really clear about default DTO value. Some people expects to have a new instance of their DTO.

The goal of this PR is to add a bit of details about how to define this default value.

see [#54042](https://github.com/symfony/symfony/issues/54042)

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
Thanks Neil!

I think this is correct, but I'll wait for other reviews before merging.

File: controller.rst
Comment (alexandre-daubois) [inline controller.rst:455]:
```suggestion
define a default value to your controller parameter::
```

File: controller.rst
Comment (alexandre-daubois) [inline controller.rst:471]: I think this line may be superfluous 🙂

**Comment by javiereguiluz** [comment]
Neil, thanks a lot for this improvement ... and congrats on your first Symfony Docs contribution 🎉

File: controller.rst
Comment (javiereguiluz) [reply controller.rst:471]: Yes, I removed it while merging. Thanks.

File: controller.rst
Comment (javiereguiluz) [reply controller.rst:455]: I tweaked this while merging ... but apparently, the correct preposition here is `for` instead of `to` 😊  Thanks!

**Pull request merged**
